pkg/fleet/installer/setup/common/config.go

```diff
@@ -168,11 +168,12 @@ type IntegrationConfig struct {
 
 // IntegrationConfigLogs represents the configuration for the logs of an integration
 type IntegrationConfigLogs struct {
-	Type    string `yaml:"type,omitempty"`
-	Path    string `yaml:"path,omitempty"`
-	Service string `yaml:"service,omitempty"`
-	Source  string `yaml:"source,omitempty"`
-	Tags    string `yaml:"tags,omitempty"`
+	Type      string `yaml:"type,omitempty"`
+	Path      string `yaml:"path,omitempty"`
+	Service   string `yaml:"service,omitempty"`
+	Source    string `yaml:"source,omitempty"`
+	Tags      string `yaml:"tags,omitempty"`
+	MultiLine bool   `yaml:"auto_multi_line_detection,omitempty"`
```

Comment:
I think it would make it more explicit to keep the exact same name as what is added in the `datadog.yaml` config

```suggestion
	AutoMultiLineDetection bool   `yaml:"auto_multi_line_detection,omitempty"`
```